EpikCord/__main__.py

```diff
@@ -1,4 +1,60 @@
-from EpikCord import __version__
+import argparse
+import colorama
+
+
+__all__ = ("__version__", "info")
+__version__ = "0.5.2"
```

Comment:
Why are you putting back version to main? and `__all__` is useless in this file

Reply:
@TheUntraceable told me long back that it is to be added

Reply:
> @TheUntraceable told me long back that it is to be added

not in `__main__.py`